Rock/MergeTemplates/HtmlMergeTemplateType.cs

```diff
@@ -61,6 +61,8 @@ public override BinaryFile CreateDocument( MergeTemplate mergeTemplate, List<obj
             BinaryFile outputBinaryFile = null;
 
             var rockContext = new RockContext();
+            rockContext.Database.CommandTimeout = 600;
```

Comment:
this shouldn't be necessary, getting a BinaryFile by Id should take less than a second